fastlane/lib/fastlane/environment_printer.rb

```diff
@@ -132,20 +132,47 @@ def self.print_system_environment
       require "openssl"
 
       env_output = "### Stack\n\n"
-      product, version, build = `sw_vers`.strip.split("\n").map { |line| line.split(':').last.strip }
+      product, version, build = "Unknown"
+      os_version = "UNKNOWN"
+
+      if FastlaneCore::Helper.mac?
+        product, version, build = `sw_vers`.strip.split("\n").map { |line| line.split(':').last.strip }
+        os_version = version
+      end
+
+      if FastlaneCore::Helper.linux?
+        # this should work on pretty much all linux distros
+        os_version = `uname -a`.strip
+        version = ""
+        build = `uname -r`.strip
+        product = `cat /etc/issue.net`.strip
+
+        # see if we can get a bit more details using 'lsb_release'
+        # check if it is availaible
+        lsb_release = `which lsb_release`.strip
```

Comment:
in fact /etc/lsb-release is not existing by default on debian only with some weird `edu-package` - and contains:

```
DISTRIB_DESCRIPTION="DebianEdu/Skolelinux"
```

see my latest commit - with a different try.